docs/using-radicle/pushing-and-pulling-changes.md

```diff
@@ -5,11 +5,11 @@ title: Pushing and pulling changes
 
 ## Committing and pushing code
 
-To push code to Radicle, you first need to make sure your shell is configured correctly by following the instructions in the [Getting Started](getting-started/doc1-1.md) section. 
+To push code to Radicle, you first need to make sure your shell is configured correctly by following the instructions in the [Getting Started](getting-started/doc1-1.md) section.
 
-After the Radicle remote helper is configured, pushing to Radicle is as simple as pushing to pushing to any other platform. After changes are added and commited, they can be pushed to the Radicle remote via `git push rad`. For help using Git, please refer to [relevant documentation](https://docs.github.com/en/free-pro-team@latest/github/using-git).
+After the Radicle remote helper is configured, pushing to Radicle is as simple as pushing to any other platform. After changes are added and commited, they can be pushed to the Radicle remote via `git push rad`. For help using Git, please refer to [relevant documentation](https://docs.github.com/en/free-pro-team@latest/github/using-git).
```

Comment:
Can we link to a resource on how to use git that is not part of the github documentation?

Reply:
Good point, it felt strange linking to GH to me as well. How about: https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup?

Reply:
:black_heart:

Reply:
Ya fine with this. Mainly had the link as a placeholder to include some git documentation.

Reply:
https://github.com/radicle-dev/radicle-docs/pull/16/commits/0a69cfa041ac1b77bfde914997b6f423d2d04864